qualification/report/generate_pdf.py

```diff
@@ -670,7 +670,7 @@ def collate_results(results: Iterable[Result]) -> list[ResultSet]:
     for result_list in result_lists.values():
         result_set = ResultSet(result_list)
         # Check whether the CBF titles alone give unique subtitles
-        cbf_titles = set(result.cbf.title for result in result_list if result.cbf is not None)
+        cbf_titles = {result.cbf.title for result in result_list if result.cbf is not None}
```

Comment:
Interesting new syntax.

Reply:
Actually that's been around a long time, although I can't immediately find which Python version introduced it.

Reply:
Looks like it dates right back to Python 3.0 (and 2.7).